node/src/main/resources/network-defaults.conf

```diff
@@ -0,0 +1,193 @@
+waves.network-name = testnet
+
+waves.defaults {
+  testnet {
+    # Blockchain type
+    blockchain.type = TESTNET
+
+    network {
+      # Port number
+      port = 6863
+
+      # List of IP addresses of well known nodes.
+      known-peers = ["52.30.47.67:6863", "52.28.66.217:6863", "52.77.111.219:6863", "52.51.92.182:6863"]
+    }
+
+    # Node's REST API settings
+    rest-api {
+      # Enable/disable node's REST API
+      enable = yes
+
+      # Port to listen to REST API requests
+      port = 6869
+    }
+  }
+
+  devnet {
+    blockchain {
+      type = CUSTOM
+      custom {
+        address-scheme-character = "D"
+        functionality {
+          feature-check-blocks-period = 5
+          blocks-for-feature-activation = 4
+          allow-temporary-negative-until = 1477958400000
+          allow-invalid-payment-transactions-by-timestamp = 1477958400000
+          require-sorted-transactions-after = 1477958400000
+          generation-balance-depth-from-50-to-1000-after-height = 0
+          minimal-generating-balance-after = 0
+          allow-transactions-from-future-until = 0
+          allow-unissued-assets-until = 1479416400000
+          require-payment-unique-id-after = 0
+          allow-invalid-reissue-in-same-block-until-timestamp = 0
+          allow-multiple-lease-cancel-transaction-until-timestamp = 1492128000000
+          reset-effective-balances-at-height = 4650
+          block-version-3-after-height = 0
+          pre-activated-features {
+            1 = 0
+            2 = 0
+            3 = 0
+            4 = 0
+            5 = 0
+            6 = 0
+            7 = 0
+            8 = 0
+            9 = 0
+            10 = 0
+          }
+          double-features-periods-after-height = 1000000000
+          max-transaction-time-back-offset = 120m
+          max-transaction-time-forward-offset = 90m
+        }
```

Comment:
add rewards section for devnet config. Compare network configs with master before move to other file